model/src/main/java/org/cbioportal/model/Mutation.java

```diff
@@ -35,10 +35,6 @@
     private String driverFilterAnnotation;
     private String driverTiersFilter;
     private String driverTiersFilterAnnotation;
-    @NotNull
-    private Boolean sequenced;
-    @NotNull
-    private Boolean wildType;
```

Comment:
why are you reverting this?

Reply:
because these values depends on the verbose gene panel data logic, which we change here, so I'll introduce them later.